components/Navbar.vue

```diff
@@ -42,6 +53,13 @@ export default {
       ],
     };
   },
+  methods: {
+    toggleNav(event) {
+      event.preventDefault();
+      event.currentTarget.classList.toggle('active');
+      this.$el.querySelector('.nav-list').classList.toggle('hide');
```

Comment:
[Refs](https://vuejs.org/v2/api/#ref) should be used here instead of `querySelector`